DeAceroDLabs/marketplace-frontend
# Feature/dmp 687 Real product to shoppingCartItems

## What is done

[[DMP-687] Pasar producto real al carrito.](https://deacero.atlassian.net/browse/DMP-687)


## How is done

- Changed type CartProduct so that it receives sku, quantity, userId, articleId too.
- Passed selectedProperties to characteristics,
- Passed price with value 1 for now, later I will delete it.
- Button with disable true if price is 0.
- Quantity is added is product is already there.

## How to test

Clear local storage:
![image](https://user-images.githubusercontent.com/81311646/200900756-33934b07-cb6f-4555-91b1-5101a5cd0759.png)


1. Run the project via `npm start`.
2. Click on any product.
3. Click `Agregar al carrito` button.
4. Click `Ver carrito` button. You should see the product you just added.
5. Items shoul be added on the cartItems table.
![image](https://user-images.githubusercontent.com/81311646/201763887-a8504200-11eb-46cd-a41d-92f3c8d3d3c6.png)


.
